### DISCUSSION TITLE
New Language (Persian)

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#1994] [General]
Hello
I'm from Iran and i want to use RustDesk recently
I want to contribute in the translation of the software in the Persian language
Thanks

**Comment by rustdesk** [comment]
Go ahead please.

**Comment by ghost** [comment]
[fa.rs.txt](https://github.com/rustdesk/rustdesk/files/9958115/fa.rs.txt)

**Reply by rustdesk** [reply]
you can put fa.rs under https://github.com/rustdesk/rustdesk/tree/master/src/lang

**Reply by ghost** [reply]
done

**Reply by rustdesk** [reply]
PR please

**Reply by ghost** [reply]
Excuse me, what do you mean by PR?

**Reply by rustdesk** [reply]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

**Reply by rustdesk** [reply]
Thanks
